fix(content): Reword phrases like “the below sections” across multiple pages (#38846)
mdn/content: files/en-us/mozilla/add-ons/webextensions/internationalization/index.md

    - locale-dependent CSS

> Let's explore each of the new features in turn — each of the sections below represents a step to follow when internationalizing your extension.

Reviewer: ```suggestion
The following sections describe the steps to take when internationalizing your extension.
```
